site/pom.xml

```diff
@@ -14,7 +14,7 @@
 
 	<properties>
 		<!-- Mars convention -->
-		<publishingPath>mars/snapshots/builds/${JOB_NAME}/${BUILD_ID}-B${BUILD_NUMBER}/all/repo/</publishingPath>
+		<publishingPath>mars/snapshots/builds/${JOB_NAME}/${BUILD_TIMESTAMP}-B${BUILD_NUMBER}/all/repo/</publishingPath>
 		<!-- Luna convention: TODO JBIDE-18772: remove this when master build only works for Mars -->
 		<publishingPath>builds/staging/${JOB_NAME}/all/repo/</publishingPath>
```

Comment:
@nickboldt I'm curious, doesn't this second definition override the first one? And if it does, why isn't anything published here? http://download.jboss.org/jbosstools/builds/staging/reddeer_master/ but this one works:
http://download.jboss.org/jbosstools/mars/snapshots/builds/reddeer_master/
and it does seem to use ${BUILD_TIMESTAMP}-B${BUILD_NUMBER} today.
Would that be because it's overridden in the job? But the job has the neon path set up, not mars:
publishingPath=neon/snapshots/builds/${JOB_NAME}/${BUILD_TIMESTAMP}-B${BUILD_NUMBER}/all/repo/
And, how does maven know about this variable if there is no -D... in the mvn deploy call?